core/src/main/java/org/elasticsearch/common/settings/PropertyPlaceholder.java

```diff
@@ -34,24 +34,13 @@
  * Values for substitution can be supplied using a {@link Properties} instance or using a
  * {@link PlaceholderResolver}.
  */
-public class PropertyPlaceholder {
+class PropertyPlaceholder {
```

Comment:
Since I can't leave emoji on a line of code without leaving a comment, this comment is so that I can: :+1: